.github/workflows/caching.yml

```diff
@@ -164,7 +167,7 @@ jobs:
           cache-name: hackage-sources
         with:
           path:  ${{ env.CABAL_PKGS_DIR }}
-          key:   ${{ env.cache-name }}-${{ env.INDEX_STATE }}-${{ hashFiles('cabal.project.freeze') }}
+          key:   ${{ env.cache-name }}-${{ env.INDEX_STATE }}-${{ steps.compute-cache-key.outputs.value }}
```

Comment:
Still, please `hackage-sources` should not depend of `freeze`, depending on `index-state` is enough for its cause.

Reply:
ah ok, so I wanted to have all consistent using one path and finally we can do it the other way around, nice anyway

Reply:
Oh - that started from my mistake. I put `-${{ hashFiles('cabal.project.freeze') }}` there in the first place :facepalm:

Reply:
fixed 😀